prow/config/config.go

```diff
@@ -30,7 +30,7 @@ import (
 
 	"github.com/ghodss/yaml"
 	"github.com/sirupsen/logrus"
-	"gopkg.in/robfig/cron.v2"
+	cron "gopkg.in/robfig/cron.v2"
```

Comment:
why?

Reply:
We've now merged this exact change already https://github.com/kubernetes/test-infra/pull/8878#discussion_r205952912
`goimports` doesn't like that this is used in the code as `cron` (I don't really either... this import is weird)

Reply:
#8878 explains this, goimports does it automatically